src/shared/containers/GetOrganizations.tsx

```diff
@@ -92,6 +92,12 @@ const GetOrganizations: FunctionComponent = () => {
     }
   }, [account.type, account.isUpgradeable])
 
```

Comment:
Do we need a redux state for whether this API call has happened? I _suspect_ this is only going to make one API because this hook is going to run only when the component mounts and when org id is populated (and org id won't change until the app reload managed by quartz when switching orgs). But it might be safer to have another condition be that the status of have fetched the notebook array is also `RemoteDataState.NotStarted`.

Might be worth testing to see how many times this API call gets made in remocal. If it's more than once, I think we need a redux state for tracking the status of the API call.

Reply:
@wdoconnell I only saw it get called once, since `org.id` hasn't changed. I didn't test it against switching orgs, but I imagine it'll get called again in that instance.

Reply:
Gotcha. As long as it's only running once, that works.